### ISSUE TITLE
Failed to start Aries Toolbox on Windows

### ISSUE THREAD

**Issue by swcurran** [hyperledger/aries-toolbox#142]
I'm running Aries Toolbox on Windows (10, git-bash) and received this error on the console and in the Aries Toolbox window.  Any ideas?

I've recently run it with the same commands on Linux.

```
 ERROR in Template execution failed: ReferenceError: process is not defined

  ERROR in   ReferenceError: process is not defined

    - index.ejs:11 eval
      [.]/[html-webpack-plugin]/lib/loader.js!./src/index.ejs:11:2

    - index.ejs:16 module.exports
      [.]/[html-webpack-plugin]/lib/loader.js!./src/index.ejs:16:3

    - index.js:284
      [aries-toolbox]/[html-webpack-plugin]/index.js:284:18

    - runMicrotasks

    - task_queues.js:97 processTicksAndRejections
      internal/process/task_queues.js:97:5
```

Commands run:

```
git clone https://github.com/hyperledger/aries-toolbox
cd aries-toolbox
npm i
npm run dev
```

```
$ npm --version
6.13.6
$ node --version
v13.7.0
```

**Comment by swcurran** [comment]
Tried this solution and it seemed to at least get the main window started. However, I don't know enough about why this would work vs. what was already there (which was a different solution):

https://github.com/SimulatedGREG/electron-vue/issues/871#issuecomment-574627835

Notably, I changed this part of `src/index.eps` to the following:

```
<!-- Set `__static` path to static files in production -->
<% if (!htmlWebpackPlugin.options.isBrowser && !htmlWebpackPlugin.options.isDevelopment) { %>
    <script>
      window.__static = require('path').join(__dirname, '/static').replace(/\\/g, '\\\\')
    </script>
<% } %>

<!-- webpack builds are automatically injected -->
```

Unfortunately, I can't go further right now as I would have to do a lot of system updates to continue. I'll try to get a Windows and Mac person to try this out.

**Comment by swcurran** [comment]
I also noticed this warning on the Windows operation:

```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
```

Not sure what the impact of not having `fsevents` available affects the operation of the app.

**Comment by dbluhm** [comment]
@TelegramSam have you run the toolbox on windows recently?

**Comment by dbluhm** [comment]
@burdettadam was this the error you were seeing that required your fix in #143?

**Comment by swcurran** [comment]
Changing Confirmed that fix worked for bringing up the main window on Windows 10.

I'll open another issue if the fsevents() thing is a big issue.

Thanks.
